### ISSUE TITLE
Duplicated error message (from PostgreSQL) in query status

### ISSUE THREAD

**Issue by trkoch** [jakob/Postico#262]
Result

```
ERROR:  duplicate key value violates unique constraint "pk_kunde"
DETAIL:  Key (kunden_nr)=(107) already exists.
Query failed
PostgreSQL said: duplicate key value violates unique constraint "pk_kunde"
Detail: Key (kunden_nr)=(107) already exists.
```

Expected (I guess)

```
Query failed
PostgreSQL said: duplicate key value violates unique constraint "pk_kunde"
Detail: Key (kunden_nr)=(107) already exists.
```

On a side note, is there a way to clear the query status?

**Comment by jakob** [comment]
Right, that is annoying, I need to fix that.

There is no way to remove query status / query results at the moment, but it sounds like a good idea. I'll add a keyboard shortcut for that.

**Comment by trkoch** [comment]
Thanks! 

FYI I tried hitting CMD+K to clear the console which is the shortcut I use in iTerm2.

**Comment by jakob** [comment]
OK. I was thinking Ctrl-L, since that's the standard unix command, but I just tried and Terminal also supports Cmd-K. Cmd-K it is, then!

**Comment by jakob** [comment]
Prerelease: https://eggerapps-downloads.s3-eu-west-1.amazonaws.com/postico-1503.zip

**Comment by trkoch** [comment]
Works perfectly! 👍

**Comment by jakob** [comment]
This fix was released in version Postico 1.0.10.
